docs/02-knora-ontologies/knora-base.md

```diff
@@ -7,605 +7,459 @@
 
 Resources can have properties that point to other resources; see
-[Links Between Resources](#links-between-resources). A resource grants permissions to groups
-of users; see [Authorisation](#authorisation).
+[Links Between Resources](#links-between-resources). A resource grants permissions to groups of users;
+see [Authorisation](#authorisation).
 
 #### Representations
 
-It is not practical to store all data in RDF. In particular, RDF is not
-a good storage medium for binary data such as images. Therefore, Knora
-stores such data outside the triplestore, in ordinary files. A resource
-can have metadata about a file attached to it. The technical term for such a
-resource in Knora is a **Representation**. For each file, there is a
-`kb:FileValue` in the triplestore containing metadata about the file
-(see [FileValue](#filevalue)). Knora uses [Sipi](https://github.com/dhlab-basel/Sipi)
-to store files. The [Knora APIs](../03-apis/index.md) provide ways
-to create file values using Knora and Sipi.
+It is not practical to store all data in RDF. In particular, RDF is not a good storage medium for binary data such as
+images. Therefore, Knora stores such data outside the triplestore, in ordinary files. A resource can have metadata about
+a file attached to it. The technical term for such a resource in Knora is a **Representation**. For each file, there is
+a `kb:FileValue` in the triplestore containing metadata about the file (see [FileValue](#filevalue)). Knora
+uses [Sipi](https://github.com/dhlab-basel/Sipi) to store files. The [Knora APIs](../03-apis/index.md) provide ways to
+create file values using Knora and Sipi.
 
 A resource that has a file value must belong to one of the subclasses of
 `kb:Representation`. Its subclasses include:
 
-`StillImageRepresentation`
-
-:   A representation containing a still image file.
-
-`MovingImageRepresentation`
-
-:   A representation containing a video file.
-
-`AudioRepresentation`
-
-:   A representation containing an audio file.
+- `StillImageRepresentation`:  
+  A representation containing a still image file.
 
-`DDDrepresentation`
+- `MovingImageRepresentation`:  
+  A representation containing a video file.
 
-:   A representation containing a 3D image file.
+- `AudioRepresentation`:  
+  A representation containing an audio file.
 
-`TextRepresentation`
+- `DDDrepresentation`:  
+  A representation containing a 3D image file.
 
-:   A representation containing a formatted text file, such as an XML file.
+- `TextRepresentation`:  
+  A representation containing a formatted text file, such as an XML file.
 
-`DocumentRepresentation`
+- `DocumentRepresentation`:  
+  A representation containing a document (such as a PDF file) that is not a text file.
 
-:   A representation containing a document (such as a PDF file) that is
-    not a text file.
+- `BundleRepresentation`:  
+  A representation containing a bundle (such as a zip archive).
```

Comment:
Wouldn't make sense to keep these  and other bullet pointes describing props as a one-liners?

Reply:
done